mail_tracking/models/mail_tracking_email.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import urlparse
+import time
+from datetime import datetime
+
+from openerp import models, api, fields, tools
+import openerp.addons.decimal_precision as dp
+
+_logger = logging.getLogger(__name__)
+
+
+class MailTrackingEmail(models.Model):
+    _name = "mail.tracking.email"
+    _order = 'time desc'
+    _rec_name = 'name'
+    _description = 'MailTracking email'
```

Comment:
I think that this is a proactive task. In other words, if you want to see what happened with one message you'll see status icon. I you want to see if any message is bounced, rejected, ... then go to settings and see las tracking emails in that state